src/modules/45-dashboards/pages/home/HomePage.tsx

```diff
@@ -333,12 +333,12 @@ const HomePage: React.FC = () => {
         />
       </Layout.Vertical>
 
-      {!loading && (
+      {!loading && !!data?.items && (
         <Layout.Vertical padding={{ left: 'medium', right: 'medium' }}>
           <Pagination
-            itemCount={dashboardList?.items || 1}
+            itemCount={data?.items || 1}
```

Comment:
You now know that this has a value, so you can use it directly
```suggestion
            itemCount={data.items}
```